app/controllers/Model.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import org.joda.time.DateTime
+import play.api.libs.json.Json
+
+case class SubmissionDetails(name: String, email: String, dateSubmitted: DateTime)
```

Comment:
change name to requesterName or submitterName ?
It looks it could be appName here